### DISCUSSION TITLE
historias de usuario - plantilla de requerimientos

### DISCUSSION THREAD

**Discussion by MacaCoronel93** [Launch-X-Latam/MisionFrontEnd#181] [Q&A]
Hola MC : 
( Se que no hay que llenar toda la plantilla de requerimientos) 
Pero cuando en la plantilla nos habla de historia de usuario y despues continua con los criterios de aceptacion --
1) Los criterios de aceptacion son con respecto a la historia de usuario en particular que se definio con anterioridad? 
2) Por cada requerimiento abria que armar una historia de usuario especifica ?

**Comment by FaraN1ght** [comment]
Según mi experiencia, las historias de usuario son situaciones en las que el usuario (según su rol admin o user) estaría en relación a la app. Los criterios de aceptación son (valga la redundancia xD) criterios que definen que una historia de usuario esté bien hecha o abarque lo que se necesita.

En la práctica de Abogabot utilicé historias de usuario y criterios de aceptación (no sé si debía pero lo hice), puedes revisar mi repositorio si te place y si puedes déjame una opinión para mejorar.

https://github.com/FaraN1ght/Abogabot/blob/main/CasoAbogabotApp.md
